### DISCUSSION TITLE
We're still processing your application. Once this is complete you will have access to the student developer pack offers

### DISCUSSION THREAD

**Discussion by GergesShamon** [community/community#147679] [GitHub Education]
### Hi everyone,

We're still processing your application. Once this is complete you will have access to the student developer pack offers.
this is the notification i'm getting more than a year ago and unable to reverify and apply for student developer pack when i try to reverify it opens pricing page please help.
I also moved from high school to university.

**Comment by TonyEinstein** [comment]
I have also encountered this problem. Would you please solve it now? How is it solved?

**Reply by GergesShamon** [reply]
Not solved yet ):

**Comment by Azzybana** [comment]
@GergesShamon I'm doing a little copy pasta to others who've been waiting or 1-3 weeks like I have, but holy crap... you posted before Xmas. I hope the necro gets you help. If not, like, you're past a month, file a ticket.

**Reply by GergesShamon** [reply]
How do I submit a ticket?
